addons/exact-count/studio.js

```diff
@@ -0,0 +1,34 @@
+export default async function({
+  addon,
+  global,
+  console
+}) {
+  function countProjects(url, offset, delta, callback) {
+    const request = new XMLHttpRequest();
+    request.open("GET", url + offset);
+    request.onreadystatechange = function() {
+      if (request.readyState == 4) {
+        let pageLen = JSON.parse(request.response).length
+        if (pageLen == 40) {
+          countProjects(url, offset + delta, delta, callback);
+        } else if (pageLen > 0) {
+          let count = 40 * (offset-1) + pageLen;

```

Comment:
Shouldn't this just be `let count = offset+1 + pageLen`?
Example, a studio has 1000 projects. You go to offset 999, it returns a project. So 999+1=1000.

Reply:
Yeah, Whoops